alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/Descriptor.java

```diff
@@ -36,6 +36,13 @@
 import com.synopsys.integration.alert.common.persistence.model.DefinedFieldModel;
 import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
 
+// Note when creating a new descriptor you will need to register descriptors in the database.
```

Comment:
Can we make this a JavaDoc comment? It will allow developers to see the comment without having to visit the file.